source/neuropod/bindings/c/c_api.h

```diff
@@ -29,19 +29,44 @@ extern "C" {
 
 typedef struct NP_Neuropod NP_Neuropod;
 
-// Load a model given a path
+// Load a model given a path.
+// status is set if user provided a valid pointer.
 void NP_LoadNeuropod(const char *neuropod_path, NP_Neuropod **model, NP_Status *status);
 
-// TODO: Add more options
+// Runtime Options. See description of options at neuropod/options.hh
 typedef struct NP_RuntimeOptions
 {
-    // Whether or not to use out-of-process execution
-    // (using shared memory to communicate between the processes)
+    // Whether or not to use out-of-process execution.
     bool use_ope;
 
+    // These options are only used if use_ope is set to true.
+    struct NP_OPEOptions
+    {
+        bool free_memory_every_cycle;
+        char control_queue_name[256];
```

Comment:
Maybe pull the 256 out into a constant?

Reply:
Added define QUEUENAME_MAX (similar to standard FILENAME_MAX)